app/src/main/java/com/onelist/dao/IUserDAO.kt

```diff
@@ -1,8 +1,8 @@
 package com.onelist.dao
 
 interface IUserDAO {
-    var firstname: String;
```

Comment:
You are able to have semi colons but it's best practice to omit them entirely. 

Link to source: https://kotlinlang.org/docs/coding-conventions.html#semicolons

Reply:
Yeah, good call.  And a hard habit to break.  :)

Until you go from Kotlin back to Java again...